share/doc/wake/logging.md

```diff
@@ -0,0 +1,65 @@
+# Logging
+
+This article gives a high level overview of how wake logging works.
+Check out [print.wake](https://github.com/sifive/wake/blob/master/share/wake/lib/core/print.wake)
+for commented code.
+
+## How Logging works on Wake Execution commands
+
+Every `Job` run by wake will have its own stdout and stderr, and when the underlying process prints to stdout or stderr, it will go the `Job`-specific stdout and stderr, not the stdout and stderr of the `wake` process itself.
```

Comment:
```suggestion
Every `Job` run by wake will have its own stdout and stderr,
and when the underlying process prints to stdout or stderr,
it will go the `Job`-specific stdout and stderr,
not the stdout and stderr of the `wake` process itself.
```

trying to be good about breaking lines on punctuation